std_msgs/Header header

uint32 seq # id of the neuro message taken into account
bool has_artifact # flag artifact presence in the current data used for the features computation